config/routes.rb

```diff
@@ -35,8 +35,10 @@
       resources :listen_signals, only: [:index, :show, :create, :update, :destroy] do
         get :templates, on: :collection
       end
-      resources :brands, only: [:show] do
+      resources :brands do
         get '/me' => 'brands#show', on: :collection
+        get '/account_plans' => 'brands#account_plans', on: :collection
+        post '/account_info' => 'brands#update_account_info', on: :collection
```

Comment:
We need to add in here a route for the account page here too. See [line 46](https://github.com/project-signal/project-signal/blob/e62571ccddd64b89678f353cc996b807d64bd722/config/routes.rb#L46) of this file.